instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -184,13 +188,27 @@ def process_exception(self, request, exception):
             return
 
         if self._environ_activation_key in request.META.keys():
+            status_code = 500
```

Comment:
Curious, the status_code always 500?

Reply:
Almost! There are some special cases where a user can raise an exception to return a specific HTTP error. I've re-implemented it in a much better way that doesn't assume status code.